.gitignore

```diff
@@ -107,4 +107,11 @@ erl_crash.dump
 # VSCode Elixir language support
 .elixir_ls/
 
+# asdf version control
+.tool-versions
```

Comment:
Eh? I don't get this leh

Reply:
woops... was supposed to not commit this, then i forgot `.gitignore` itself gets committed ._.